### PR TITLE
May - July 2021 Roadmap

### PR DESCRIPTION
**Pull request by maryelizbeth**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Updating the roadmap for May through July, moving February-April roadmap into `docs/roadmaps` archive.

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,102 +1,92 @@
...
+We select items for inclusion in the roadmap from the Top 10 Community Issues, [Core Services](docs/CORE_SERVICES.md), and internal priorities. Where community sourced contributions exist we will work with the authors to review and merge their work. Where this does not exist or the original contributors are not available we will create the resources and implementation ourselves.
 
 Each weekly release will include necessary tasks that lead to the completion of the stated goals as well as community pull requests, enhancements, and features that are not highlighted in the roadmap. To view all the items we've prioritized for this quarter, please see the [Roadmap milestone](https://github.com/hashicorp/terraform-provider-aws/milestone/138).
 
 This roadmap does not describe all the work that will be included within this timeframe, but it does describe our focus. We will include other work as events occur .
 
-From [November through January](docs/roadmaps/2020_November_to_January.md), we added support for (among other things):
+In February 2020 to April 2021 846 Pull Requests were opened in the provider and we merged 947(as of April 13th), adding support for:
```
Comment (breathingdust) [inline ROADMAP.md:11]:
```suggestion
In the period spanning February to April 2021 846 Pull Requests were opened in the provider and 947 were merged, adding support for:
```

File: docs/roadmaps/2021_February_to_April.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+### CloudWatch Synthetics
+Issue: [#11145](https://github.com/hashicorp/terraform-provider-aws/issues/11145)
+
+_[CloudWatch Synthetics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries.html) You can use Amazon CloudWatch Synthetics to create canaries, configurable scripts that run on a schedule, to monitor your endpoints and APIs. Canaries follow the same routes and perform the same actions as a customer, which makes it possible for you to continually verify your customer experience even when you don't have any customer traffic on your applications. By using canaries, you can discover issues before your customers do._
+
+Support for CloudWatch Synthetics will include:
+
+New Resource(s):
+- aws_synthetics_canary
```
Comment (breathingdust) [inline docs/roadmaps/2021_February_to_April.md:48]:
```suggestion
New Resource(s):

- aws_synthetics_canary
```

File: docs/roadmaps/2021_February_to_April.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+New Datasource(s):
+- aws_synthetics_canary_runs
```
Comment (breathingdust) [inline docs/roadmaps/2021_February_to_April.md:51]:
```suggestion
New Datasource(s):

- aws_synthetics_canary_runs
```

File: docs/roadmaps/2021_February_to_April.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+- Gateway Load Balancer
+- Spot Launch for EKS Managed Node Groups
+
+From February-April ‘21, we will be prioritizing the following areas of work:
+
+## Provider Functionality: Default Tags
+
+Issue: [#7926](https://github.com/hashicorp/terraform-provider-aws/issues/7926)
+
+Default Tags builds on the workflows in Ignore Tags to provide additional control over the ways Terraform manages tagging capabilities. Users will be able to specify lists of tags to apply to all resources in a configuration at the provider level. Our goal in offering this use case is to assist in tidying up configuration files, decreasing development efforts, and streamlining cost allocation and resource attribution within organizations of all sizes.
```
Comment (breathingdust) [inline docs/roadmaps/2021_February_to_April.md:36]:
```suggestion
Default Tags builds on the workflows in Ignore Tags to provide additional control over the ways Terraform manages tagging capabilities. Users will be able to specify lists of tags to apply to all resources in a configuration at the provider level. Our goal in offering this use case is to assist in tidying up configuration files, decreasing development efforts, and streamlining cost allocation and resource attribution within organizations of all sizes.
```

File: docs/roadmaps/2021_February_to_April.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
+# Roadmap:  February 2021 - April 2021
+
+Every few months, the team will highlight areas of focus for our work and upcoming research.
+
+We select items for inclusion in the roadmap from the Top 10 Community Issues, [Core Services](docs/CORE_SERVICES.md), and internal priorities. Where community sourced contributions exist we will work with the authors to review and merge their work. Where this does not exist or the original contributors, are not available we will create the resources and implementation ourselves.
```
Comment (ewbankkit) [inline docs/roadmaps/2021_February_to_April.md:5]: The link has to be relative to the referencing page's directory.

```suggestion
We select items for inclusion in the roadmap from the Top 10 Community Issues, [Core Services](../CORE_SERVICES.md), and internal priorities. Where community sourced contributions exist we will work with the authors to review and merge their work. Where this does not exist or the original contributors, are not available we will create the resources and implementation ourselves.
```

File: docs/roadmaps/2021_February_to_April.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+Each weekly release will include necessary tasks that lead to the completion of the stated goals as well as community pull requests, enhancements, and features that are not highlighted in the roadmap. To view all the items we've prioritized for this quarter, please see the [Roadmap milestone](https://github.com/hashicorp/terraform-provider-aws/milestone/138).
+
+This roadmap does not describe all the work that will be included within this timeframe, but it does describe our focus. We will include other work as events occur .
+
+From [November through January](docs/roadmaps/2020_November_to_January.md), we added support for (among other things):
```
Comment (ewbankkit) [inline docs/roadmaps/2021_February_to_April.md:11]: The link has to be relative to the referencing page's directory.

```suggestion
From [November through January](2020_November_to_January.md), we added support for (among other things):
```

**Pull request merged**